conformance/lang/expressions/shift-expression/left_shift.balt

```diff
@@ -0,0 +1,3816 @@
+    //        = In decimal: (-9223372036854775808)
+
+    io:println(b << c); // @output 63
+    // Explanation:
+    // Value to be shifted = Equivalent binary representation : 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00111111
+    // Shift amount = Before applying mask, In binary: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 01000000, In decimal: 64
+    //                After applying mask, In binary: 000000, In decimal: 0
+    // Result = In binary: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00111111
+    //        = In decimal: (63)
+
+    io:println(a << e); // @output -9223372036854775808
+    // Explanation:
+    // Value to be shifted = Equivalent binary representation : 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111
+    // Shift amount = Before applying mask, In binary: 11111111 11111111 11111111 11111111 11111111 11111111 11111111 10111111, In decimal: -65
+    //                After applying mask, In binary: 111111, In decimal: 63
+    // Result = In binary: 10000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+    //        = In decimal: (-9223372036854775808)
+
+    io:println(a << f); // @output -1
+    // Explanation:
+    // Value to be shifted = Equivalent binary representation : 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111
+    // Shift amount = Before applying mask, In binary: 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11000000, In decimal: -64
+    //                After applying mask, In binary: 000000, In decimal: 0
+    // Result = In binary: 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111
+    //        = In decimal: (-1)
+
+    io:println(a << a); // @output -9223372036854775808
+    // Explanation:
+    // Value to be shifted = Equivalent binary representation : 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111
+    // Shift amount = Before applying mask, In binary: 11111111 11111111 11111111 11111111 11111111 11111111 11111111 11111111, In decimal: -1
+    //                After applying mask, In binary: 111111, In decimal: 63
+    // Result = In binary: 10000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+    //        = In decimal: (-9223372036854775808)
+
+    io:println(e << c); // @output -65
+    // Explanation:
+    // Value to be shifted = Equivalent binary representation : 11111111 11111111 11111111 11111111 11111111 11111111 11111111 10111111
+    // Shift amount = Before applying mask, In binary: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 01000000, In decimal: 64
+    //                After applying mask, In binary: 000000, In decimal: 0
+    // Result = In binary: 11111111 11111111 11111111 11111111 11111111 11111111 11111111 10111111
+    //        = In decimal: (-65)
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int' operands.
```

Comment:
Let's be more specific
How about ```Test resultant type of the left shift expression being `int` with 'int' operands via invalid assignments.```

Reply:
+1. Will update. Thanks.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-spec/pull/953/commits/2fe94debf0c3d87472283ccb4c9b99ae3142eeb9.